visualization/app/codeCharta/codeCharta.model.ts

```diff
@@ -194,7 +195,38 @@ export interface MetricData {
 	maxValue: number
 }
 
+export interface CCLocalStorage {
+	version: string
+	scenarios: [String, RecursivePartial<Scenario>][]
+}
+
 export interface Scenario {
+	name: string
+	area: {
+		areaMetric: string
+		margin: number
+	}
+	height: {
+		heightMetric: string
+		heightSlider: number
+		labelSlider: number
+	}
+	color: {
+		colorMetric: string
+		colorRange: ColorRange
+	}
+	camera: {
+		camera: Vector3
+		cameraTarget: Vector3
+	}
+	edge: {
+		edgeMetric: string
+		edgePreview: number
+		edgeHeight: number
+	}
+}
+
+export interface ScenarioAsSettings {
```

Comment:
ScenariosAsSettings is our Scenario-API for the cc.json, right?

If that is the case, let's rename it to ExportScenario. We should continuously divide our interfaces into external (cc.json) and internal (codecharta application). This will improve our type-safety and will help a lot, if we do some api changes at some point